### PR TITLE
docs: Add notes on how to compile a dynamic module for nginx

### PR DESCRIPTION
**Pull request by nikolas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,13 +24,21 @@ Apache. As a result, this current version has less dependencies, fewer bugs, and
 Before compile this software make sure that you have libmodsecurity installed.
 You can download it from the [ModSecurity git repository](https://github.com/SpiderLabs/ModSecurity). For information pertaining to the compilation and installation of libmodsecurity please consult the documentation provided along with it.
 
-With libmodsecurity installed, you can proceed with the installation of the ModSecurity-nginx connector, which follow the nginx 3rd party module installation procedure:
+With libmodsecurity installed, you can proceed with the installation of the ModSecurity-nginx connector, which follows the nginx third-party module installation procedure. From the nginx source directory:
 
 ```
-./configure --add-module=/path/to/ModSecurity-nginx
+./auto/configure --add-module=/path/to/ModSecurity-nginx
```
Comment (dvershinin) [inline README.md:30]: The vast majority of source downloads of NGINX are the tarball from their official website, which has no file `auto/configure`. The `./configure` should be kept as it is more common (this is what's present in the official tarball).

File: README.md [hunk 2]
```diff
@@ -24,13 +24,21 @@ Apache. As a result, this current version has less dependencies, fewer bugs, and
...
 ```
 
-Further information about nginx 3rd party add-ons support are available here:
+Or, to build a dynamic module:
+
+```
+./auto/configure --add-dynamic-module=/path/to/ModSecurity-nginx <nginx_configure_flags>
+```
+
+Note that when building a dynamic module, your nginx source version and configure flags need to match. You can get both of these with `nginx -V`.
```
Comment (dvershinin) [inline README.md:39]: The configure arguments don't have to match. Just add `--with-compat` switch.

File: README.md (hunk 1 above)
Comment (nikolas) [reply README.md:30]: Oh - thx. Yeah, I downloaded from GitHub, but I agree this should match whatever the source download tarball from their site has.

**Comment by zimmerle** [comment]
thank you @dvershinin and @nikolas

**Pull request merged**